### ISSUE TITLE
Suggestion: Allow password update for basic authentication

### ISSUE THREAD

**Issue by ivanayov** [openfaas/faas#833]
Forgetting the password for basic-auth starts to be a common issue. (#823, #825)

There is a [documentation](https://docs.openfaas.com/deployment/troubleshooting/#i-forgot-my-gateway-password) for how to retrieve the old password, but people might find it hard to follow every time they forget it. Password update would allow them to set their own.

**Comment by alexellis** [comment]
To address this I added a third method for password retrieval via looking at .openfaas/config.yml

I'll close this issue, but feel free to track suggestions here.
